pkg/subcmd/config.go

```diff
@@ -169,6 +177,32 @@ func (c *Config) runCreate() error {
 	return err
 }
 
+// runDelete controls the deletion process.
+func (c *Config) runDelete() error {
+	if c.flags.DryRun {
```

Comment:
What about say It would delete CM config in namespace RHTAP or something like that, basically tell the user what it would do if dry run was not enabled

Reply:
Sure! Based on what we've spoken, please consider the latest changes.